shared.c

```diff
@@ -542,11 +543,11 @@ runImmediately
     trSetObscuredUnits(false);
 	trSetCivAndCulture(1, 9, 3); // Set P1 to Kronos
 	trSetCivAndCulture(2, 9, 3); // Set P2 to Kronos
-	bool multiplayer = aiIsMultiplayer(); 	// nottud is smart
-	if(multiplayer && kbIsPlayerHuman(2) == false){
-		multiplayer = false; // or kick?
+	bool Multiplayer = aiIsMultiplayer(); 	// nottud is smart
```

Comment:
wait doesn't this break? variable was declared outside the trigger already

Reply:
no but done